### ISSUE TITLE
Ability to dismiss "Notifications are disabled" warning

### ISSUE THREAD

**Issue by jeremiah-k** [binwiederhier/ntfy#1220]
:bulb: **Idea**
It's not a serious issue, but it would be nice to be able to hide the "Notifications are disabled" warning in the webapp.

I often use the ntfy web app to manually send notifications to other users and am not interested in enabling notifications in my browser on certain machines.

:computer: **Target components**
web app

**Comment by unoukujou** [comment]
I agree there needs to be a dismiss button for people who don't want this. "Grant Permission or look at this forever" is definitely undesirable.

The message is punchy. Takes up valuable screen real estate. It's right in the face. It doesn't get out of the way. It's annoying ;)

Not everybody wants desktop notifications so there has to be some way to dismiss this message. The browser normally handles this natively with a pop-up that lets you know, and allows you to dismiss it forever for "this site". That should be the way to do it, in my opinion.

Possibly it can be move to the "/settings" page. I would say that's where it belongs.
